antelope_contracts/contracts/erc20/src/erc20.cpp

```diff
@@ -477,4 +477,51 @@ inline eosio::name erc20::receiver_account()const {
     return get_self();
 }
 
+void erc20::callupgrade(std::string proxy_address) {
+    require_auth(get_self());
+
+    config_t config = get_config();
+
+    auto address_bytes = from_hex(proxy_address);
+    eosio::check(!!address_bytes, "token address must be valid 0x EVM address");
+    eosio::check(address_bytes->size() == kAddressLength, "invalid length of token address");
+
+    checksum256 addr_key = make_key(*address_bytes);
+    token_table_t token_table(_self, _self.value);
+    auto index = token_table.get_index<"by.address"_n>();
```

Comment:
This is fine. but seems we would be better to use the "native contract + symbol" lookup. (similar to unregtoken & setegressfee)